CruCentralCoast/Profile/ChooseCampusOrMovement/ChooseCampusVC.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  ChooseCampusVC.swift
+//  CruCentralCoast
+//
+//  Created by Michael Cantrell on 4/26/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class ChooseCampusVC: UITableViewController {
+    
+    var subscribedMovements = [String]()
+    var shouldReloadSelectedIndexPath = false
+    var selectedIndexPath: IndexPath?
+    
+    private let searchController = UISearchController(searchResultsController: nil)
+    
+    private var campuses: Results<Campus>!
+    private var filteredCampuses = [Campus]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "Choose Campus"
+        self.configureTableView()
+        self.configureSearchController()
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(self.doneButtonPressed))
+        
+        self.subscribedMovements = LocalStorage.preferences.getObject(forKey: .subscribedMovements) as? [String] ?? []
+        
+        DatabaseManager.instance.subscribeToDatabaseUpdates(self)
+        self.campuses = DatabaseManager.instance.getCampuses()
+        let _ = DatabaseManager.instance.getMovements()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        // If a campus with multiple movements has had its subscription changed, update the cell
+        if shouldReloadSelectedIndexPath, let indexPath = self.selectedIndexPath {
```

Comment:
`self.shouldReload`

Reply:
This was the best name I could think of

Reply:
I was just saying to add the `self.` to `shouldReloadSelectedIndexPath`